notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Instead of focusing on "a dataframe has attributes", I would maybe rather start with explaining the concepts of column names and index. And then only after that explain that those two "things" are attributes ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>left discussion on attributes of a tabel out of the starting tutorial to descrease technical elements</p>